doc/cci_guide.md

```diff
@@ -0,0 +1,1703 @@
+        negative of cci_set_autocommit                                  negative of cci_set_autocommit
+
+        negative of cci_get_autocommit                                  negative of cci_get_autocommit
+        test.c(82) - batch_01 ERROR : Invalid connection handle ( [0]   test.c(82) - batch_01 ERROR : Invalid connection handle ( [0]
+
+        test.c(88) - batch_01 ERROR : Invalid connection handle ( [0]   test.c(88) - batch_01 ERROR : Invalid connection handle ( [0]
+
+        autocommit: 0                                                   autocommit: 0
+        num_query=0 of cci_execute_batch()                              num_query=0 of cci_execute_batch()
+        num_query<0 of cci_execute_batch()                              num_query<0 of cci_execute_batch()
+        conn_handle is null in cci_execute_batch()                      conn_handle is null in cci_execute_batch()
+        test.c(122) - batch_01 ERROR : Invalid connection handle)       test.c(122) - batch_01 ERROR : Invalid connection handle) 
+
+        conn_handle is digit in cci_execute_batch()                     conn_handle is digit in cci_execute_batch()
+        num_query< number_sql_stmts of cci_execute_batch()              num_query< number_sql_stmts of cci_execute_batch()
+
+        positive cases                                                  positive cases
+
+        query_result: 0@0@1@1@1@1@                                      query_result: 0@0@1@1@1@1@
+
+        negative of cci_end_tran()                                      negative of cci_end_tran()
+        test.c(179) - batch_01 ERROR : Invalid transaction type argum   test.c(179) - batch_01 ERROR : Invalid transaction type argum
+
+        test.c(185) - batch_01 ERROR : Invalid connection handle)       test.c(185) - batch_01 ERROR : Invalid connection handle) 
+
+        test.c(191) - batch_01 ERROR : Invalid connection handle)       test.c(191) - batch_01 ERROR : Invalid connection handle) 
+
+        $ echo $?
+        0
+        ```
+       If the .output file is different from .answer file, the result turns out NOK.
+
+# 3. Deploy Regression Tests Environment
+## 3.1 Test Machine
+**Controller node**: It listens to test messages and starts a test when there is a test message. It will distribute test cases to each test node for execution.  
+**Test node**: It executes test cases.
+For current daily regression test, controller node and test node are the same one.  
+<table>
+  <tr>
+    <th>No.</th>
+    <th>Role</th>
+    <th>User Name</th>
+    <th>IP</th>
+    <th>Hostname</th>
+  </tr>
+   <tr>
+    <th>1.</th>
+    <th>Controller node and Test node</th>
+    <th>cci</th>
+    <th>192.168.1.78</th>
+    <th>func03</th>
+  </tr>
+</table>
+
+## 3.2 Deploy Tests Environment
+### On controller & test node
+* Install CTP
+  - Please follow [install CTP](#21-install-ctp).  
+    Configure CTP/conf/common.conf as below:
```

Comment:
Remove content "Configure CTP/conf/common.conf as below"

Replace with:
Please refer to the guide to [install CTP as Regression Test platform](https://github.com/CUBRID/cubrid-testtools/blob/develop/doc/ctp_install_guide.md#3-install-ctp-as-regression-test-platform).
https://github.com/CUBRID/cubrid-testtools/blob/develop/doc/ctp_install_guide.md#3-install-ctp-as-regression-test-platform